buffalo/algo/als.py

```diff
@@ -140,7 +140,7 @@ def _iterate(self, buf, group='rowwise'):
             f'{group} updated: processed({updated}) elapsed(data feed: {feed_t:0.3f}s update: {update_t:0.03}s)')
         return loss_nume, loss_deno
 
-    def train(self):
+    def train(self, training_callback=None):
```

Comment:
`training_callback`에 대한 어노테이션이 있으면 좋을것 같습니다.

```suggestion
from typing import Callable, Optional, Dict


    def train(self, training_callback: Optional[Callable[[int, Dict[str, float]], None]] = None):
```